### ISSUE TITLE
Use pre-trained weights on custome dataset

### ISSUE THREAD

**Issue by Gaurav2912** [Deci-AI/super-gradients#1086]
### 💡 Your Question

Hi! I trained YOLO-NAS on custom dataset but my notebook crashed, now I want to train it again, but I want to use latest weights (ckpt_best.pth) . How shold I do.. hear I use 

from super_gradients.training import models
model = models.get('yolo_nas_l', 
                   num_classes=len(dataset_params['classes']), 
                   pretrained_weights= "coco"
                   )

It take pretrained_weights= "coco" as arguments . how to use my checkpoints/my_first_yolonas_run/ckpt_best.pth as weight for train the model again.

**Comment by Rohinivv96** [comment]
@Gaurav2912 Use the following command 
yolo detect train resume model=/content/runs/detect/train/weights/last.pt
and also ensure that you have a complete runs folder and a custom dataset folder.

**Comment by harpreetsahota204** [comment]
Hi @Gaurav2912, you can follow the documentation for `Loading model Weights from a Checkpoint` [here](https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/Checkpoints.md#loading-model-weights-from-a-checkpoint).
